### DISCUSSION TITLE
Run Anthias on Raspberry Pi Zero 2 W?

### DISCUSSION THREAD

**Discussion by Timodg** [Screenly/Anthias#1760] [Q&A]
Hi,

I have a couple of Raspi Zero 2 W's and would like to install the software on those devices. 
I run into a lot of problems while installing through the install link, beside that the disk images seem not to work for the Zero 2 W.

Could you optimize the image for the Zero 2 W and which image is meant to use for this raspberry device?

Please let me know if you need additional information.

Kind regards,
Timo

**Comment by vpetersson** [comment]
There's plenty of optimization that can (and will) be done in the coming months. We're in a big overhaul phase right now where we are moving the code base to Python 3 (and Django). Once we've finished this, we'll be looking into optimizing various parts of the system to reduce both the docker image sizes and CPU/memory overhead.

**Reply by Timodg** [reply]
Good to hear, I understand the phase where you are currently in. I'm looking forward to all the improvements 

To answer my question, with which image should the Zero 2 W be installed, the image for the pi3?

**Reply by vpetersson** [reply]
You're probably best off installing it manually on Raspbian for now. That way you can disable services by hand if need be.

**Comment by Timodg** [comment]
Okay, I run into a couple of errors while using the manual install, is there anything I can try to fix this? 
See image in the attachment
![IMG_0845](https://user-images.githubusercontent.com/30000178/222134972-3e8005bb-5542-4c41-8468-cfad4e7ff326.JPG)

**Reply by vpetersson** [reply]
Try installing Docker by hand.

**Reply by Timodg** [reply]
It seems after installing Docker by hand, the problem still occurs. Do you maybe have some additional solutions?

**Reply by vpetersson** [reply]
We don't have a workaround for this. We'll be prioritizing getting the Balena version working first.

**Comment by Timodg** [comment]
First docker manual install, then manual install Anthias?
